lib/src/components/NxSidebarNavigation/NxSidebarNavitation.scss

```diff
@@ -0,0 +1,184 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+@import '../../scss-shared/nx-container-helpers';
+
+$nx-sidebar-width__open: 250px;
+$nx-sidebar-width__closed: 69px;
```

Comment:
Values like `69px` and `33px` seem very _magical_

Reply:
> Values like `69px` and `33px` seem very _magical_

I got them directly out of what Zeplin indicated. Should I define them in `_nx-variables.scss` or ... ?

Reply:
> > Values like `69px` and `33px` seem very _magical_
> 
> I got them directly out of what Zeplin indicated. Should I define them in `_nx-variables.scss` or ... ?

We have been using a base-8 grid for most of our spacing so I would have expected the sizing to embrace that.

Reply:
> > > Values like `69px` and `33px` seem very _magical_
> > 
> > 
> > I got them directly out of what Zeplin indicated. Should I define them in `_nx-variables.scss` or ... ?
> 
> We have been using a base-8 grid for most of our spacing so I would have expected the sizing to embrace that.

A lot of the dimensions in the mock are definitely _not_ standard. That includes margins/paddings and widths.
@stephenim ^^

Reply:
> That includes margins/paddings and widths.

Width and height often won't be; they usually depend on content.  It's mainly about padding and margins.